service/src/test/java/org/sunbird/learner/actors/client/ClientManagementActorTest.java

```diff
@@ -1,192 +1,233 @@
+import static org.powermock.api.mockito.PowerMockito.when;
 
 import akka.actor.ActorRef;
 import akka.actor.ActorSystem;
 import akka.actor.Props;
 import akka.testkit.javadsl.TestKit;
+import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import org.apache.commons.lang3.StringUtils;
 import org.junit.*;
-import org.junit.runners.MethodSorters;
-import org.sunbird.cassandra.CassandraOperation;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.sunbird.actor.router.RequestRouter;
+import org.sunbird.actor.service.BaseMWService;
+import org.sunbird.actorutil.InterServiceCommunicationFactory;
+import org.sunbird.cassandraimpl.CassandraOperationImpl;
+import org.sunbird.common.ElasticSearchUtil;
 import org.sunbird.common.exception.ProjectCommonException;
 import org.sunbird.common.models.response.Response;
 import org.sunbird.common.models.util.ActorOperations;
 import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.datasecurity.DecryptionService;
 import org.sunbird.common.request.Request;
-import org.sunbird.common.responsecode.ResponseMessage;
+import org.sunbird.common.responsecode.ResponseCode;
 import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.actors.role.dao.impl.RoleDaoImpl;
 import org.sunbird.learner.util.Util;
-
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
-@Ignore
+import org.sunbird.user.dao.impl.UserOrgDaoImpl;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  ServiceFactory.class,
+  RoleDaoImpl.class,
+  BaseMWService.class,
+  RequestRouter.class,
+  InterServiceCommunicationFactory.class,
+  ElasticSearchUtil.class,
+  Util.class,
+  UserOrgDaoImpl.class,
+  DecryptionService.class,
+})
+@PowerMockIgnore({"javax.management.*"})
 public class ClientManagementActorTest {
 
-  private static ActorSystem system;
+  private static ActorSystem system = ActorSystem.create("system");;
   private static final Props props = Props.create(ClientManagementActor.class);
-  private static String masterKey = "";
-  private static String clientId = "";
-
-  @BeforeClass
-  public static void setUp() {
-    system = ActorSystem.create("system");
-    Util.checkCassandraDbConnections(JsonKey.SUNBIRD);
+  private static String masterKey = "anyMasterKey";
+  private static String clientId = "anyClientId";
+  private static String clientName = "anyClientName";
+  private static String channel = "anyChannel";
+  private static String id = "anyId";
+  private static CassandraOperationImpl cassandraOperation = mock(CassandraOperationImpl.class);
+  private static Map<String, Object> request;
+  private static Map<String, Object> orgMap;
```

Comment:
this is always overridden in Before test - this should not be static in that case.

Reply:
simply initialize here with constructor